appmap/_implementation/testing_framework.py

```diff
@@ -147,3 +147,14 @@ def collect_result_metadata(metadata):
         metadata["test_status"] = "failed"
         metadata["exception"] = {"class": exn.__class__.__name__, "message": str(exn)}
         raise
+
+def file_write(filename, contents):
+    f = open(filename, "w")
+    f.write(contents)
+    f.close()
+
+def file_delete(filename):
+    try:
+        os.remove(filename)
+    except:
```

Comment:
This bare except hides the error that `os` hasn't been imported. It should just be catching `FileNotFoundError`, and allowing any other exception to propagate.